govtool/frontend/Dockerfile

```diff
@@ -1,4 +1,6 @@
 FROM node:18-alpine as builder
+
+ARG VITE_APP_ENV='beta'
```

Comment:
Why you are setting here a default value? Is there a reason?

Reply:
In case someone forgets to provide build arg